igor-web/src/main/groovy/com/netflix/spinnaker/igor/jenkins/JenkinsBuildMonitor.groovy

```diff
@@ -78,7 +78,7 @@ class JenkinsBuildMonitor extends CommonPollingMonitor {
     void poll() {
         log.info "- Polling cycle started - ${new Date()}"
         buildMasters.filteredMap(BuildServiceProvider.JENKINS).keySet().parallelStream().forEach(
-            { master -> changedBuilds(master) }
+            { master -> processProjectsOfMaster(master) }
```

Comment:
Did we rename in the Monitor interface? not seeing it in this changeset

Reply:
The monitor interface doesn't really define much.

Reply:
What Rob said, this is not part of the interface